rotkehlchen/chain/evm/decoding/thegraph/decoder.py

```diff
@@ -58,6 +63,51 @@ def _decode_delegator_staking(self, context: DecoderContext) -> DecodingOutput:
             return self._decode_stake_locked(context)
         elif context.tx_log.topics[0] == TOPIC_STAKE_DELEGATED_WITHDRAWN:
             return self._decode_stake_withdrawn(context)
+        elif context.tx_log.topics[0] == DELEGATION_TRANSFERRED_TO_L2:
+            return self._decode_delegation_transferred_to_l2(context)
+        return DEFAULT_DECODING_OUTPUT
+
+    def _decode_delegation_transferred_to_l2(self, context: DecoderContext) -> DecodingOutput:
+        """Decode a transfer delegating GRT from ethereum to arbitrum"""
+        delegator_l2 = hex_or_bytes_to_address(context.tx_log.topics[2])
+        indexer = hex_or_bytes_to_address(context.tx_log.topics[3])
+        l2_indexer = hex_or_bytes_to_address(context.tx_log.data[:32])
+        transferred_delegation_tokens = token_normalized_value(
+            token_amount=hex_or_bytes_to_int(context.tx_log.data[32:]),
+            token=self.token,
+        )
+        event = self.base.make_event_from_transaction(
+            transaction=context.transaction,
+            tx_log=context.tx_log,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=self.token,
+            balance=Balance(amount=FVal(transferred_delegation_tokens)),
+            notes=f'Delegation of {transferred_delegation_tokens} GRT transferred from indexer {indexer} to L2 indexer {l2_indexer}.',  # noqa: E501
+            location_label=context.transaction.from_address,
+            counterparty=CPT_THEGRAPH,
+            address=context.transaction.to_address,
+            extra_data={'delegator_l2': delegator_l2},
+        )
+        return DecodingOutput(event=event)
+
+    def _decode_contract_deposit(self, context: DecoderContext) -> DecodingOutput:
+        """Decode a deposit of ETH to cover the arbitrum fees of delegating GRT"""
+        indexer = hex_or_bytes_to_address(context.tx_log.topics[1])
```

Comment:
same here. You should check if the address in question is tracked. Otherwise we don't care about it.